apps/src/code-studio/progress.js

```diff
@@ -197,7 +197,10 @@ function getLevelProgress(signedIn, progressData, scriptName) {
     case null:
       // We do not know if user is signed in or not, send a request to the server
       // to find out if the user is signed in and retrieve progress information
-      return $.ajax(`/api/user_progress/${scriptName}`)
+      return $.ajax({
+        url: `/api/user_progress/${scriptName}`,
+        data: {user_id: clientState.queryParams('user_id')}
```

Comment:
It's interesting we have several ways to get url params, another way that I've been using is just importing the `queryParams` method from https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/utils.js#L26

Reply:
I've been replacing with `queryParams`, just because it's shorter. I don't have a strong preference, it's already inconsistent so 🤷‍♀️ , feel free to leave as-is